portal/templates/portal/teach/dashboard.html

```diff
@@ -154,11 +154,15 @@ <h3>These teachers have requested to join your school or club</h3>
                                     {{ join_request.new_user.last_name }}</small></p></td>
                                 <td><p><small>{{ join_request.new_user.email }}</small></p></td>
                                 <td class="cell-center">
-                                    <a id="allow_button" class="button button--small button-primary button--primary--positive"
-                                            href="{% url 'organisation_allow_join' join_request.id %}">Allow</a></td>
+                                     <button id="allow_button" class="button button--small button-primary button--primary--positive"
+                                            onclick="post('{% url 'organisation_allow_join' join_request.id %}', {
```

Comment:
Question as I'm not sure exactly what's happening here, can this post be now replaced with`postWithCsrf()` if we decide to use it (or post with a param if you accept my suggestion)?

Reply:
yup - 
here i think postWithCsrf might prove more concise